server/src/test/java/io/crate/metadata/functions/SignatureBinderTest.java

```diff
@@ -558,6 +559,9 @@ public void testNumericParameters() {
         assertThatSignature(NumericSumAggregation.SIGNATURE)
             .boundTo(dt)
             .hasReturnType(dt);
+        assertThatSignature(NumericMinAggregation.SIGNATURE)
+            .boundTo(dt)
+            .hasReturnType(dt);
```

Comment:
Can we add the same for max?

Reply:
Oh, I've added it already, but forgot to push, i had it squashed to the 1st commit. pushed the change.